.changeset/hip-walls-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+"@thirdweb-dev/storage": patch
+---
+
+moved res.json() to avoid getting JSON error in console
```

Comment:
changesets should ideally be useful for end-users of the packages

Reply:
Got it. Did it in a hurry, will keep in mind. Thanks